Sources/PrimerSDK/Classes/Core/3DS/3DSService.swift

```diff
@@ -413,7 +413,7 @@ class ThreeDSService: ThreeDSServiceProtocol {
                     // All good, url value is valid and https
                     threeDsAppRequestorUrl = url
                 } else {
-                    print("PRIMER WARNING!\nthreeDsAppRequestorUrl is not in a valid format (\"https://applink\"). In case you want to support redirecting back during the OOB flows please set correct threeDsAppRequestorUrl in PrimerThreeDsOptions during SDK initialization.")
+                    logger.warn(message: "Warning: threeDsAppRequestorUrl is not in a valid format (\"https://applink\"). In case you want to support redirecting back during the OOB flows please set correct threeDsAppRequestorUrl in PrimerThreeDsOptions during SDK initialization.")
```

Comment:
```suggestion
                    logger.warn(message: "threeDsAppRequestorUrl is not in a valid format (\"https://applink\"). In case you want to support redirecting back during the OOB flows please set correct threeDsAppRequestorUrl in PrimerThreeDsOptions during SDK initialization.")
```

sorry, I meant this :)